README.md

```diff
@@ -153,6 +153,29 @@ var options = {
 app.use('/api-docs', swaggerUi.serve, swaggerUi.setup(null, options));
 ```
 
+To load multiple swagger documents from urls as a dropdown in the explorer bar, pass an array of object with `name` and `url` to 'swaggerUrls' property of the options to the setup function.
+
+```javascript
+const express = require('express');
+const app = express();
+const swaggerUi = require('swagger-ui-express');
+
+var options = {
+  swaggerUrls: [
+    {
+      url: 'http://petstore.swagger.io/v2/swagger.json',
+      name: 'Spec1'
+    },
+    {
+      url: 'http://petstore.swagger.io/v2/swagger.json',
+      name: 'Spec2'
+    }
+}
```

Comment:
My mental model is that top-level options affect the`swagger-ui-express` layer, and options within `swaggerOptions` affect the underlying UI. 

With that in mind, I think it'd make more sense to do this through `swaggerOptions` and preserve the UI's `urls` naming convention, i.e.:

```js
var options = {
  swaggerOptions: {
    urls: [
      {
        url: 'http://petstore.swagger.io/v2/swagger.json',
        name: 'Spec1'
      },
      {
        url: 'http://petstore.swagger.io/v2/swagger.json',
        name: 'Spec2'
      }
    ]
  }
}
```